### DISCUSSION TITLE
New Glitch Suggestions

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [Glitchy-Scratch/glitchy-scratch.github.io#17] [Ideas]
Suggest New Glitches Below!

**Comment by R4356th** [comment]
I am planning to mess up with the text and add newly added website strings and features. Do you have any suggestion about that? And by the way, I moved this to Ideas because Q&A is for questions.

**Comment by TheSmallyDog** [comment]
What about adding a glitch into the editor, like the vector sprites and vector backgrounds will sometimes turn into gray boxes with question mark.
